sphinx/ext/autodoc/__init__.py

```diff
@@ -418,9 +419,12 @@ def format_signature(self):
         else:
             return ''
 
-    def add_directive_header(self, sig):
-        # type: (str) -> None
+    def add_directive_header(self, sig, additional_options=None):
+        # type: (str, list) -> None
```

Comment:
`List[str]`?

Reply:
Rendered obsolete due recent changes in the diff.